ca_description/urdf/create_base.xacro

```diff
@@ -54,6 +55,16 @@
     <xacro:create_wheel prefix="right" y_offset="${wheel_separation / -2}" wheel_radius="${wheel_radius}" wheel_width="${wheel_width}"/>
     <xacro:caster_wheel/>
 
+    <xacro:light_sensor prefix="left">
+      <origin xyz="0.095 0.14 0.065" rpy="0 ${pi/4} 0"/>
```

Comment:
Since you're sending the origin block, do you need the `prefix`?

Reply:
I think the prefix helps in case that you want to see only one of the two frames in Gazebo or RViz